doc/howto/storage_volumes.md

```diff
@@ -131,6 +156,40 @@ All I/O limits only apply to actual block device access.
 Therefore, consider the file system's own overhead when setting limits.
 Access to cached data is not affected by the limit.
 
+````
+```` {group-tab} UI
+To attach a storage volume to an instance, navigate to the configuration page of the target instance.
+
+In the secondary menu, select {guilabel}`Disk` under {guilabel}`Devices`.
+
+From here, select {guilabel}`Attach disk device`.
+
+```{figure} /images/storage/storage_volumes_attach_to_instance_1.png
+:width: 80%
+:alt: Attach a storage volume to an instance - Disk configuration page
+```
+
+The resulting modal will allow you to choose your disk type. To attach a new custom volume, select {guilabel}`Attach custom volume`.
```

Comment:
```suggestion
The resulting modal will allow you to choose your disk type, select {guilabel}`Attach custom volume`.
```
This can be more concise. The label explains, probably no need to repeat it.